src/main/java/com/kodilla/ecommercee/repository/UserRepository.java

```diff
@@ -13,6 +14,9 @@
     @Override
     Optional<User> findById(Long id);
 
-    User findByUserName(String userName);
+    @Override
+    List<User> findAll();
+
+    Optional<User> findByUserName(String userName);
```

Comment:
@Jola-Witaszak dodałem Optionala w findByUsername, bo co jeśli nie ma usera o tai mimieniu :) 
+ dodałem nadpisnie findAll, żeby zsracało listę  List<User> (domyślnie jest Iterable<T>)

Reply:
Optionala wcale byc nie musi, można to w inny sposób obsłużyć. Dla mnie optionale są kłopotliwe w użytkowaniu i wolę je omijać

Reply:
Jak dla mnie optional w takim przypadku jest ok